.github/actions/csv-vulnerability-filter/src/types.ts

```diff
@@ -0,0 +1,29 @@
+// .github/actions/csv-vulnerability-filter/src/types.ts
+
+// Represents a row parsed from the CSV
+// Using Record<string, string> for flexibility
+export type CsvRow = Record<string, string>;
```

Comment:
If you want you could define this as a type with an `ID` field and any number of other keys with string values. I would recommend that if you know there will always be an ID field